Loop/Managers/DeviceDataManager.swift

```diff
@@ -761,12 +763,16 @@ extension DeviceDataManager: CGMManagerDelegate {
     }
 }
 
-extension DeviceDataManager: CGMManagerSetupViewControllerDelegate {
-    func cgmManagerSetupViewController(_ cgmManagerSetupViewController: CGMManagerSetupViewController, didSetUpCGMManager cgmManager: CGMManagerUI) {
+extension DeviceDataManager: CGMManagerCreateDelegate {
+    func cgmManagerCreateNotifying(_ notifying: CGMManagerCreateNotifying, didCreateCGMManager cgmManager: CGMManagerUI) {
         self.cgmManager = cgmManager
     }
 }
 
+extension DeviceDataManager: CGMManagerOnboardDelegate {
+    func cgmManagerOnboardNotifying(_ notifying: CGMManagerOnboardNotifying, didOnboardCGMManager cgmManager: CGMManagerUI) {}
+}
```

Comment:
[nit] I didn't see this in your other PR, but should`(_ notifying: CGMManagerOnboardNotifying: ...` be replaced with `( _ notifier: CGMManagerOnboardNotifier: ...`. At least to me, it helps connect that this is the notifier of that event.

Reply:
I'm just following the existing `CompletionNotifying` name scheme.